descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Finance Funds",
+  "requires": [],
+  "provides": [
+    {
+      "id": "finances",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/funds",
+          "permissionsRequired": ["finance-funds.collection.get"]
```

Comment:
Please follow the naming convention for all permissions https://dev.folio.org/guidelines/naming-conventions/
It should be like finance.funds.collection.get